Jenkinsfile

```diff
@@ -155,11 +155,11 @@ pipeline {
         string(name: 'TestTag',
                defaultValue: '',
                description: 'Test-tag to use for this run (i.e. pr, daily_regression, full_regression, etc.)')
-        string(name: 'TestNvme',
-               defaultValue: '',
-               description: 'The launch.py --nvme argument to use for the Functional test ' +
-                            'stages of this run (i.e. auto, auto_md_on_ssd, auto:-3DNAND, ' +
-                            '0000:81:00.0, etc.)')
+        // string(name: 'TestNvme',
+        //        defaultValue: '',
+        //        description: 'The launch.py --nvme argument to use for the Functional test ' +
+        //                     'stages of this run (i.e. auto, auto_md_on_ssd, auto:-3DNAND, ' +
+        //                     '0000:81:00.0, etc.).  Does not apply to MD on SSD stages.')
```

Comment:
Is this meant to be removed in the master branch?

Reply:
Yes, with there being separate MD on SSD stages that have this value fixed to auto_md_on_ssd it is not really needed for the other stages and could end up running duplicate stages with the same --nvme setting if set to auto_md_on_ssd.

Reply:
Should the commented out code just be removed?